pkg/mover/chart.go

```diff
@@ -135,13 +138,13 @@ func (cm *ChartMover) WithRetries(retries uint) *ChartMover {
 // the expected rewrites in the Helm Chart.
 func (cm *ChartMover) Print() {
 	log := cm.logger
-	log.Println("Image moves:")
+	log.Println("Image copies:")
```

Comment:
changed move from copies since that's what we technically do. We do not execute any destructive operation